pkg/sql/show_create_clauses.go

```diff
@@ -108,7 +109,11 @@ func showComments(
 
 	if tc.comment != nil {
 		buf.WriteString(";\n")
-		buf.WriteString(fmt.Sprintf("COMMENT ON TABLE %s IS '%s'", table.Name, *tc.comment))
+		buf.WriteString(fmt.Sprintf(
+			"COMMENT ON TABLE %s IS %s",
+			table.Name,
```

Comment:
Woah woah there. table.Name is a string and needs to be escaped too.

Maybe pass `tree.Name(table.Name)` 

Ditto column and index names below.

Also add tests.